src/main/scala/dpla/ingestion3/preMappingReports/XmlShredder.scala

```diff
@@ -20,17 +20,17 @@ object XmlShredder {
     * Examples:
```

Comment:
Was there a find/replace in here that went awry? I thought we were using . as a path separator and _ for separating xml element names from namespace nicknames.

Reply:
Yeah, I got it mixed up when I wrote the JSON shredder, and then couldn't recall which one was the original.  It's consistent now with both shredders.  Do you have strong feelings about which is path separator and which is element separator?

Reply:
Nope. Let's let Gretchen tell us.